### ISSUE TITLE
Invalid AWS Access Key ID

### ISSUE THREAD

**Issue by liam-ot** [bboure/serverless-appsync-simulator#188]
**Describe the bug**
[AWS has updated their access key ID naming pattern](https://repost.aws/articles/ARc4hEkF9CRgOrw8kSMe6CwQ/troubleshooting-the-access-key-id-or-security-token-is-invalid-error-after-upgrading-dynamodb-local-to-version-2-0-or-greater) which results in a breaking change when using VTL templates, serverless-offline, and this repo

I would be happy to open a PR but i dont know how to, i have the repo locally and the changes in a branch but i cannot push, instead git suggests i fork and push to that, is that the correct way to do it?

**To Reproduce**
Use a VTL template

**Expected behavior**
The VTL template should complete the request

**Additional context**
I have a branch locally with the fix, it is two lines in the index.js file where the underscores are removed from the access key and secret.

**Comment by domdomegg** [comment]
> git suggests i fork and push to that, is that the correct way to do it?

Yes, I suspect this is the correct way to go about making a contribution.

These two docs articles may be relevant:
- https://docs.github.com/en/get-started/quickstart/fork-a-repo#propose-changes-to-someone-elses-project
- https://docs.github.com/en/get-started/quickstart/contributing-to-projects#forking-a-repository
